source/simulator/core.cc

```diff
@@ -1043,6 +1046,7 @@ namespace aspect
             {
               coupling[x.velocities[d]][x.pressure] = DoFTools::always;
               coupling[x.pressure][x.velocities[d]] = DoFTools::always;
+              coupling[x.pressure][x.pressure] = DoFTools::always;
```

Comment:
Oh, that is true. No, there is no way to access them currently. We could add this to the signal. A better option might be to also compute the ``current_constraints`` before constructing the matrices. That way you don't need to change the signals.